5calls/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/activity_main"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/background_color"
-    tools:context="org.a5calls.android.a5calls.controller.MainActivity"
-    >
+<FrameLayout android:id="@+id/activity_main"
+             xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:tools="http://schemas.android.com/tools"
+             android:layout_width="match_parent"
+             android:layout_height="match_parent"
+             android:background="@color/background_color"
+             tools:context="org.a5calls.android.a5calls.controller.MainActivity">
+
+    <android.support.v4.widget.SwipeRefreshLayout
+        android:id="@+id/swipeContainer"
```

Comment:
I've been doing XML IDs like swipe_container instead of swipeContainer. Does this way work better with butterknife? Or can we stick with the other convention?